lib/jxl/enc_frame.cc

```diff
@@ -1291,6 +1291,12 @@ Status EncodeFrame(const CompressParams& cparams_orig,
     lossy_frame_encoder.State()->shared.image_features.noise_params =
         SimulatePhotonNoise(ib.xsize(), ib.ysize(), cparams.photon_noise_iso);
   }
+  if (cparams.manual_noise.size() > 0) {
```

Comment:
Couldn't this segfault with wrong lut size?

Reply:
Yes. Not anymore.